ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -221,10 +221,14 @@ export class EvalCommand extends Command<EvalCommandReturn> {
         const evaluatedValue = current.result.value;
 
         if (!ConfigSchema.CFGU.TESTS.VAL_TYPE[cfgu.type]({ ...cfgu, value: evaluatedValue })) {
+          let suggestion = `value "${evaluatedValue}" must be a "${cfgu.type}"`;
```

Comment:
I appreciate the feedback but for the sake of clarity a different error message feels necessary. "must be a ${pattern}" is misleading because the value must in reality match the pattern which emphasizes that they need to adjust the value to fit the RegEx.